webapp/src/Components/Replay/ReplayViewer/ThreeScene.jsx

```diff
@@ -38,27 +54,128 @@ export class ThreeScene extends Component {
             this.frameId = requestAnimationFrame(this.animate)
         }
     };
+
     stop = () => {
         cancelAnimationFrame(this.frameId)
     };
+
     animate = () => {
-        this.cube.rotation.x += 0.01;
-        this.cube.rotation.y += 0.01;
+        this.updateBall();
+        this.updatePlayers();
+        this.updateCamera();
+
         this.renderScene();
         this.frameId = window.requestAnimationFrame(this.animate)
     };
+
     renderScene = () => {
         this.renderer.render(this.scene, this.camera)
     };
 
+    generatePlayfield = () => {
```

Comment:
We might want the field code to a different file too since that can change depending on the "mode"

But this will also be simpler when loading models